RUFAS/routines/animal/life_cycle/animal_base.py

```diff
@@ -9,6 +9,7 @@
 from RUFAS.routines.animal.ration.amino_acid import EssentialAminoAcidRequirements
 from RUFAS.routines.animal.life_cycle.lactation_curve import LactationCurve
 from RUFAS.time import Time
+from ....enums import AnimalCombination
```

Comment:
```suggestion
from RUFAS.enums import AnimalCombination
```
Relative pathing should be avoided if not at the same level or one higher, i.e. `from .enums ... ` or `from ..enums ...` would be fine but `from ...enums ...` should be turned into a full path instead. I am still not 100% sure about the best practices related to relative pathing or the logic underlying these guidelines, but this is what I have been told.